pymoab-visit-py2-18.04

```diff
@@ -0,0 +1,28 @@
+FROM svalinn/pymoab-py2-18.04:latest
+
+ENV HOME /root
+
+RUN apt-get install -y --fix-missing \
+    wget \
+    cpio \
+    libpcre3-dev \
+    libgl1-mesa-glx \
+    libgl1-mesa-dev \
+    libsm6 \
+    libxt6 \
+    libglu1-mesa
+
+RUN pip install xmldiff
+
+RUN cd $HOME/opt \
+    && wget https://github.com/visit-dav/visit/releases/download/v3.1.1/visit3_1_1.linux-x86_64-ubuntu18.tar.gz \
+    && wget https://github.com/visit-dav/visit/releases/download/v3.1.1/visit-install3_1_1 \
+    && echo 1 > input \
+    && bash visit-install3_1_1 3.1.1 linux-x86_64-ubuntu18 /usr/local/visit < input \
```

Comment:
what is that for ?
why not just `bash visit-install3_1_1 3.1.1 linux-x86_64-ubuntu18 /usr/local/visit 1` ?

Reply:
In the visit install script, it waits for user input so without the `<` it just cycles infinitely waiting for the user. If that makes sense?

Reply:
yes it does